department-of-veterans-affairs/caseflow
# Resolve issues with converting central office hearings between virtual type

Resolves #14786 

### Description
Resolves bugs with converting hearing type between central and virtual, see the ticket for details on which bugs are resolved by this PR.

### Acceptance Criteria

### Testing Plan
1. Shadow user `BVASYELLOW`
1. Navigate to the hearing schedule and select a Central hearing: http://localhost:3000/hearings/schedule
1. Click the 'Edit Hearing Details' link
1. Change the type to Virtual
1. Go through the list of items in the ticket and ensure all of the issues are resolved
